src/e2e/pages/landingPage.js

```diff
@@ -72,9 +87,21 @@ export class LandingPage {
     await this.firefoxLogo.click()
   }
 
+  async heroSectionTexts(){
```

Comment:
This looks like something that's more appropriate in a test, rather than in the page object? In any case, there's no reason for the method to be `async`.